visualization_scripts/README.md

```diff
@@ -0,0 +1,26 @@
+## Visualization scripts
+**Requirements**
+
+Cytoscape v3.7.1
+
+Anaconda 
+
+**Setup**
+
+Before running the visualization script, edit the run.py file and within the main function. Edit `DIRNAME` and insert the directory that Cytoscape can be found at on your machine. If you do not know where `Cytoscape.exe` is located, you can choose to search the highest directory availible. The script is set to traverse you machines file system starting in the Users directory. 
```

Comment:
If we keep this approach where the user specifies the Cytoscape directory, we can make the `DIRNAME` a command like argument to `run.py`.  Let's wait to make that change until we finalize how to specify the Cytoscape path.

In general, it can be easier to track changes in the readme files by adding a newline character after each sentence.  So
```
Before running the visualization script, edit the run.py file and within the main function.
Edit `DIRNAME` and insert the directory...
```

That way git tracks edits to individual sentences instead of entire paragraphs.